### PR TITLE
New rpc

### PR DESCRIPTION
**Pull request by VladSemenyuk**

### CODE REVIEW

File: src/components/can_cooperation/src/can_module.cc [hunk 1]
```diff
@@ -80,15 +78,27 @@ void CANModule::SubscribeOnFunctions() {
...
+  plugin_info_.mobile_function_list.push_back(
+      MobileFunctionID::GET_INTERIOR_VEHICLE_DATA_CAPABILITIES);
+  plugin_info_.mobile_function_list.push_back(
+      MobileFunctionID::GET_INTERIOR_VEHICLE_DATA);
+  plugin_info_.mobile_function_list.push_back(
+      MobileFunctionID::SET_INTERIOR_VEHICLE_DATA);
+  plugin_info_.mobile_function_list.push_back(
+      MobileFunctionID::ON_INTERIOR_VEHICLE_DATA);
+
+    plugin_info_.hmi_function_list.push_back(
```
Comment (bonewell) [inline src/components/can_cooperation/src/can_module.cc]: wrong indent

**Comment by bonewell** [comment]
Reviewed.

**Comment by pvysh9** [comment]
Code doesn't compile:
[ 94%] /home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/src/mobile_command_factory.cc:48:76: fatal error: can_cooperation/commands/on_interior_vehicle_data_notification.h: No such file or directory
 #include "can_cooperation/commands/on_interior_vehicle_data_notification.h"
                                                                            ^
compilation terminated.

**Comment by pvysh9** [comment]
cpplint errors:
/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/src/command/get_interior_vehicle_data_capabilities_request.cc:43:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/src/command/get_interior_vehicle_data_capabilities_request.cc:48:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
Done processing /home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/src/command/get_interior_vehicle_data_capabilities_request.cc


/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/button_press_request.h:66:  Add #include <string> for string  [build/include_what_you_use] [4]
Done processing /home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/button_press_request.h
Total errors found: 1


/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/get_interior_vehicle_data_capabilities_request.h:53:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/get_interior_vehicle_data_capabilities_request.h:66:  Add #include <string> for string  [build/include_what_you_use] [4]
Done processing /home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/get_interior_vehicle_data_capabilities_request.h
Total errors found: 2


/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/get_interior_vehicle_data_request.h:53:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/get_interior_vehicle_data_request.h:66:  Add #include <string> for string  [build/include_what_you_use] [4]
Done processing /home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/get_interior_vehicle_data_request.h
Total errors found: 2


/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/set_interior_vehicle_data_request.h:53:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
/home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/set_interior_vehicle_data_request.h:66:  Add #include <string> for string  [build/include_what_you_use] [4]
Done processing /home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/include/can_cooperation/commands/set_interior_vehicle_data_request.h
Total errors found: 2

Please fix

**Comment by pvysh9** [comment]
Reviewed

**Comment by VladSemenyuk** [comment]
Code doesn't compile:
[ 94%] /home/dev/Projects/Ford/revsdl/sdl_core/src/components/can_cooperation/src/mobile_command_factory.cc:48:76: fatal error: can_cooperation/commands/on_interior_vehicle_data_notification.h: No such file or directory
#include "can_cooperation/commands/on_interior_vehicle_data_notification.h"
^
compilation terminated.

Unexpected CTRL+z in project explorer)))

**Pull request merged**